source/Nevermore/Diagnostics/ITransactionLogger.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Nevermore.Diagnostics
+{
+    public interface ITransactionLogger
+    {
+        void Write(long duration, string transactionName);
```

Comment:
While this could be another method in `IQueryLogger` which would make things a lot simpler in the short term, a separate interface feels cleaner and easier to work with in the future.